user-guide/Reference/DataMiner_Tools/Spectrum_Simulation_Tool.md

```diff
@@ -6,38 +6,14 @@ uid: Spectrum_Simulation_protocol
 
-> You can download this protocol from [DataMiner Dojo](https://community.dataminer.services/download/spectrum-simulation-driver/).
-
 To use this protocol:
 
-1. Upload the protocol in the *Protocols & Templates* app in DataMiner Cube.
-
-1. Create an element that uses this protocol.
-
-The element has the following data pages:
-
-- *Spectrum Analyzer*: Displays the standard Spectrum Analysis user interface.
-
-- *General*: Contains general information, as well as the following parameters:
-
-  - *Track Parameter Sets*: Open the button next to this parameter to open a tracking window. This window allows you to trace which parameter sets were done. In the window, sets will only be logged if the *Track Sets* parameter is enabled. If *Track Status Sets* is also enabled, status sets (communication between SLSpectrum and the protocol, e.g. requesting a new trace) are also logged.
-
-  - *Trace pattern*: Allows you to select a specific trace pattern instead of the default pattern. The following patterns are available:
+1. Deploy the protocol from the [DataMiner Catalog](https://catalog.dataminer.services/details/6f33ec9f-e83d-49d5-8f85-87ad66eaa5c7).
 
-    - *Default*: A single flat line at the center of the trace window, simulating no available signals or carriers.
+   > [!TIP]
+   > See also: [Deploying a Catalog item to your system](xref:Deploying_a_catalog_item)
 
-    - *Jump Up*: A single spike at the center of the window, wider than the channel pattern. This can be used to simulate e.g. a band pass filter.
-
-    - *Jump Down*: Inverse pattern of *Jump Up*. This can be used to simulate e.g. a notch filter.
-
-    - *Channel*: A single spike in the center of the trace window, simulating a typical view of one channel.
-
-    - *Band*: A set of 8 spikes spread over the trace window, simulating a band with several channels.
-
-  - *Floor*: A single line in the lower part of the trace window, simulating no available signals or carriers.
-
-  - *Noise*: Allows you to disable noise in the simulated pattern.
-
-  - *Noise depth*: Allows you to modify the amount of random noise added to the simulated pattern, so that you can for instance simulate environments with poor reception.
+1. Create an element that uses this protocol.
 
-  - *Mirror Values*: This page shows the current values of the spectrum interfacing parameters, and allows you to change them as if they were changed directly on the device. Note that the protocol will automatically sync start/stop frequency or center frequency/frequency span if one of these settings is changed.
+> [!NOTE]
+> For detailed information about this protocol, see [Skyline Spectrum Simulator](https://docs.dataminer.services/connector/doc/Skyline_Spectrum_Simulation.html).
```

Comment:
There is actually in the Driver Help still a small mistake. The driver will call Skyline Spectrum Simulation and not 'Simulator', hence in this line 19, we need to put [Skyline Spectrum Simulation] (the url is already fine)